corrscope/renderer.py

```diff
@@ -64,6 +65,8 @@
 # matplotlib.use() only affects pyplot. We don't use pyplot.
 import matplotlib
 import matplotlib.colors
+import matplotlib.pyplot as pyplot
```

Comment:
ehhh...

- usually abbreviated `plt`
- i used to import pyplot. i deliberately removed it in favor of creating a canvas myself.
- please use `matplotlib.image.imread` instead.